.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Description
+
+What changed? Why? (the code + comments should speak for itself on the "how")
+
+Fun screenshots ir a cool video or something are super helpful as well. If this touches platform-specific behavior, this is where test evidence should be collected.
+
+Any issues this pull request closes or pull requests this supersedes should be linked with `Closes #issuenumber`.
+
+## Meta
```

Comment:
what does this mean

Reply:
Just like anything that's to help people write pull requests (like the checklist isn't a part of the MR description body or anything)